examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairAddressPaseCommand.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ *   Copyright (c) 2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package com.matter.controller.commands.pairing
+
+import chip.devicecontroller.ChipDeviceController
+import com.matter.controller.commands.common.CredentialsIssuer
+import java.util.logging.Level
+import java.util.logging.Logger
+
+class PairAddressPaseCommand(controller: ChipDeviceController, credsIssue: CredentialsIssuer?) : PairingCommand(
+  controller,
+  "address-paseonly",
+  PairingModeType.ADDRESS_PASE_ONLY,
+  PairingNetworkType.NONE,
+  credsIssue
+) {
+  private val logger = Logger.getLogger(PairAddressPaseCommand::class.java.name)
```

Comment:
This should be at the top-level or in a companion object (effectively static) rather than an instance property.